.gitignore

```diff
@@ -1,4 +1,5 @@
-.coverage
+.coverage*
```

Comment:
Why do you change this? Won't this now accidentally match `.coveragerc`?

Reply:
Gooooood point... I changed it because the current setup creates a load of files called things like `.coverage.holden.local.57837.664190`. The problem is, I copied this line from our main repo. Maybe we should add `!.coveragerc` to both.

Reply:
Could we do `.coverage.*`?